### PR TITLE
generate include.js more frequently

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@callahad - mind reviewing?

generate include.js more frequently:
- On every compress
- Every time run_locally.js is called.

Also:
- Build include.js into the resources/static/build directory like other generated resources.
- Rename generate_include.js to create_include.js for consistency
- remove resources/static/include_js/include.js from the .gitignore list

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,7 +12,6 @@
 /resources/static/common/js/lib/bidbundle.js
 /resources/static/common/js/lib/gobbledygook.js
 /resources/static/dialog/views/site
-/resources/static/include_js/include.js
```
Comment (callahad) [inline .gitignore:15]: Why? Oh, because it doesn't exist anymore?

**Comment by callahad** [comment]
r+, merge on monday. Awesome!

File: .gitignore (hunk 1 above)
Comment (shane-tomlinson) [inline .gitignore:15]: Exactly. If it does, I'd like to know about it. ;)

**Comment by jaredhirsch** [comment]
merging now that the train's been branched :beers:

**Comment by lloyd** [comment]
nit: missing a bug with rationale!  this would help me write the changelog

**Comment by jaredhirsch** [comment]
for context see shane's comments in #3056: basically include.js was only getting rebuilt when postinstall.js was run. In order to make life better when hacking locally, this pull ensures that include.js is regenerated more often.

**Pull request merged**